src/mender-update/inventory_test.cpp

```diff
@@ -96,7 +99,8 @@ exit 0
 	NoAuthHTTPClient client {client_config, loop};
 
 	const string expected_request_data =
-		R"([{"name":"key1","value":["value1","value11"]},{"name":"key2","value":"value2"},{"name":"key3","value":"value3"}])";
+		R"([{"name":"key1","value":["value1","value11"]},{"name":"key2","value":"value2"},{"name":"key3","value":"value3"},{"name":"mender_client_version","value":["additional_version",")"
```

Comment:
Why are we submitting multiple? :thinking:

Reply:
Honestly I don't really have a usecase for this. But it was the only way I could think of which doesn't silently hide the existing value if it has also been provided by a script. Most likely if you end up in this situation, you've done something wrong, but this way you will at least be able to see that.

Reply:
It's still redundant. I don't think it should be doing it at all.

If two different scripts provided the value, then we'd also make an array, so this is consistent with that.

Reply:
:+1:  Alright. Let's hope we don't mess something up :pinched_fingers: